webroot/results/admin/export_public.php

```diff
@@ -157,6 +154,10 @@ function exportPublic ( $sources ) {
         echo '.';  # shows both Apache and the user that the script is doing stuff and not hanging
       }
     }
+    //Check if any errors while fetching data
+    if (($error = mysql_error()) !== '') {
```

Comment:
@FatBoyXPC You SHOULD not run `mysql_error` without any mysql connection. The only case if return false is there's no mysql connection.